geometric/tests/test_run_json.py

```diff
@@ -69,7 +69,7 @@ def test_convert_constraint_dict_full():
     }
     constraint_string = geometric.run_json.make_constraints_string(constraint_dict)
     assert constraint_string == """$freeze
-xyz 1 2 3 4 5
```

Comment:
This is actually a good example of the bug. The reference string here can not be correctly parsed by geomeTRIC.